### PR TITLE
Header and content section

### PR DESCRIPTION
**Pull request by Yacoubou-seidou**
- Header section is completed.
- Content section is completed.
- Linter test passed successfully.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+<body>
+  <div class="container">
+  <header>
+  <h3>My Logo</h3>
+  <i class="fa-solid fa-bars fa-2x icon"></i>
+  </header>
```
Comment (Abenezer-Tilahun) [inline index.html:21]: - [ ] For making Your HTML code semantic correct I recommend you to make the `header`  tag outside of the `body`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+<body>
+  <div class="container">
+  <header>
+  <h3>My Logo</h3>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] For making Your HTML code semantic correct I recommend you to use `a` tag for logo , This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  <i class="fa-solid fa-bars fa-2x icon"></i>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] For making Your HTML code semantic correct I recommend you to use `nav` tag for the hamburger button, This is for the `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      LET’S CONNECT
+    </h5>
+    <ul>
+      <li><i class="fa-brands fa-twitter socialMedia"></i></li>
+      <li><i class="fa-brands fa-linkedin-in socialMedia"></i></li>
+      <li><i class="fa-solid fa-m socialMedia"></i></li>
+      <li><i class="fa-brands fa-github socialMedia"></i></li>
+      <li><i class="fa-regular fa-hand-peace socialMedia"></i></li>
+    </ul>
+  </section>
```
Comment (Abenezer-Tilahun) [inline index.html:41]: - [ ] As stated in the project requirement, Kindly I recommend you to use  `flexbox` In order to lay out the elements on the page for headline section.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .socialMedia {
+    font-size: 22px;
+    color: #505f79;
+    font-weight: 500;
+  }
+}
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ ] As stated the project requirement Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust the overlapping between the hamburger menu and background image .

The Figma design Look likes this : 
 
Capture

Your Project Look likes this : 
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Yacoubou-seidou     , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git Hub-Flow.
✔️ Good commit messages
✔️ Professional README file.

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Yacoubou-seidou) [reply index.html:41]: The content are already align in the right way so should i use flexbox anyway? @Abenezer-Tilahun

File: index.html (hunk 3 above)
Comment (Yacoubou-seidou) [reply index.html]: @Abenezer-Tilahun  for the hamburger button should I create and hamburger menu or should I use an icon I don't understand the project requirement.

File: style.css (hunk 5 above)
Comment (Yacoubou-seidou) [reply style.css]: @Abenezer-Tilahun i made all the changes can you please give me more advices

File: index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+  crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+<body>
+    <header>
+      <a>My Logo</a>
```
Comment (mahbubul14) [inline index.html]: - [ ] You have used `a` tag but your text is not linkable. You can get an idea from the following example of how to do it.

`Ex: <a href="https://www.w3schools.com">My Logo</a>`

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Yacoubou-seidou,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 6 above)
Comment (Yacoubou-seidou) [reply index.html]: @mahbubul14  should the link take to some address?

**Review by Ibizugbe — APPROVED** [review]
Hi @Yacoubou-seidou 👋, 

JalenHurtsPhiladephiaEaglesGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅

- [ ] Github flow practice has been used👍
- [ ] Linters are green👍

## Optional suggestions

- [ ] [OPTIONAL] The link to the logo of your readme file is currently broken, this is giving your readme file a less professional look😉 I suggest you fix the issue🤗

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**